statsmodels/graphics/tsaplots.py

```diff
@@ -201,10 +202,10 @@ def seasonal_plot(grouped_x, xticklabels, ylabel=None, ax=None):
     ticks = []
     for season, df in grouped_x:
         df = df.copy() # or sort balks for series. may be better way
-        if pandas.__version__ < '0.17.0':
-            df.sort()
-        else:
+        if is_pandas_min_version(StrictVersion('0.17.0')):
```

Comment:
IMO this block is not the right way to handle this.  There should be a funciton in sm.compat.pandas called `sort_values` that would be something like

```
def sort_values(df, *args):
   if PD17:
       return df.sort_values(*args)
    else:
        return df.sort(*args)
```
There might be more mapping needed if kwargs changes location or names, although I don't think they did.  The point of `sm.compat` is to avoid any dependency-version-specific stuff in main code.

This also makes it easier in the long run since the function can be removed form `sm.compat.pandas` after the min version is >= 0.17 and then any imports of `sm.compat.pandas` can be removed.